common/namespace/namespace.go

```diff
@@ -89,6 +100,10 @@ func FromPersistentState(record *persistence.GetNamespaceResponse) *Namespace {
 		isGlobalNamespace:           record.IsGlobalNamespace,
 		failoverNotificationVersion: record.Namespace.FailoverNotificationVersion,
 		notificationVersion:         record.NotificationVersion,
+		customSearchAttributesMapper: CustomSearchAttributesMapper{
```

Comment:
It is every 10s for every namespace. Just FYI.

Reply:
Hm. Maybe follow up optimization to avoid calling this if there are no changes.